src/core/packet.c

```diff
@@ -595,12 +595,12 @@ QuicPacketLogHeader(
         case QUIC_VERSION_VER_NEG: {
             QuicTraceLogVerbose(
                 LogPacketVersionNegotiation,
-                "[%c][%cX][-] VerNeg DestCid:%s SrcCid:%s (Payload %lu bytes)",
+                "[%c][%cX][-] VerNeg DestCid:%s SrcCid:%s (Payload %u bytes)",
                 PtkConnPre(Connection),
                 (uint8_t)PktRxPre(Rx),
                 QuicCidBufToStr(DestCid, DestCidLen).Buffer,
                 QuicCidBufToStr(SourceCid, SourceCidLen).Buffer,
-                (uint16_t)(PacketLength - Offset));
+                (uint32_t)(PacketLength - Offset));
```

Comment:
Let's just replace `Payload %lu bytes` with `Payload %hu bytes` and keep it as a `uint16_t`.